epp-client/src/tests/de.rs

```diff
@@ -302,59 +302,59 @@ mod response {
         let ns_list = result.info_data.ns.as_ref().unwrap();
         let ns = (*ns_list).host_obj.as_ref().unwrap();
         let hosts = result.info_data.hosts.as_ref().unwrap();
+        let statuses = result.info_data.statuses.as_ref().unwrap();
```

Comment:
If the previous commit doesn't pass tests without this commit, these should probably be squashed?

Reply:
Fair point. Squashed.